setup.cfg

```diff
@@ -1,55 +1,4 @@
-[metadata]
-name = aiohttp
-version = attr: aiohttp.__version__
-url = https://github.com/aio-libs/aiohttp
-project_urls =
-  Chat: Matrix = https://matrix.to/#/#aio-libs:matrix.org
-  Chat: Matrix Space = https://matrix.to/#/#aio-libs-space:matrix.org
-  CI: GitHub Actions = https://github.com/aio-libs/aiohttp/actions?query=workflow%%3ACI
-  Coverage: codecov = https://codecov.io/github/aio-libs/aiohttp
-  Docs: Changelog = https://docs.aiohttp.org/en/stable/changes.html
-  Docs: RTD = https://docs.aiohttp.org
-  GitHub: issues = https://github.com/aio-libs/aiohttp/issues
-  GitHub: repo = https://github.com/aio-libs/aiohttp
-description = Async http client/server framework (asyncio)
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-maintainer = aiohttp team <team@aiohttp.org>
-maintainer_email = team@aiohttp.org
-license = Apache-2.0 AND MIT
```

Comment:
This SPDX identifier got lost somewhere.

Reply:
@cdce8p have you re-checked if this affects the metadata after all the updates applied recently?